### DISCUSSION TITLE
is iOS still supported after the removal of the Dylib engine as of version 3?

### DISCUSSION THREAD

**Discussion by Connoropolous** [wasmerio/wasmer#3449] [General]
I am aware that the engines got consolidated into one engine as of v3, but I noticed that the integration tests for iOS are commented out since that PR. Is there still any support for iOS?

**Comment by ptitSeb** [comment]
Hi,

Yes, for now, creating a Dylib for iOS require many manual step, and so the iOS tests were commented out. iOS is not the current focus of the developpement team, but the idea of supporting iOS device is not dropped.

**Reply by Connoropolous** [reply]
Thanks for the reply @ptitSeb . Can you expand on this at all? I'd like to understand a bit better. 
The wasmer website cites iOS as one of the target platforms. What level of supporting that platform is wasmer currently targeting?

**Reply by Amosel** [reply]
Hope it is ok to piggyback on this comment, I would also be interested knowing if you open to considering future enhancements and work.
